### ISSUE TITLE
MC.13.1 + S:T323I, S:L368I, S:T572I, ORF1a:T614I, ORF1b:G219S (23 in Canada)

### ISSUE THREAD

**Issue by Fearuncle** [sars-cov-2-variants/lineage-proposals#2237]
![image](https://github.com/user-attachments/assets/7356d850-14ca-48d5-9e41-9214e0335d7d)
tree:https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_a7f4_edab70.json?f_userOrOld=uploaded%20sample

First Seq: 2024-10-28
Most Recent: 2024-11-09

Gisaid: G14122A, C22664A, A26040C

EPI_ISL_19538097 EPI_ISL_19551630 EPI_ISL_19551629 EPI_ISL_19538523 EPI_ISL_19538801 EPI_ISL_19538650

**Comment by FedeGueli** [comment]
Yes this has been highlighted by me in the groups some days ago with the query: G14122A, C22664A, A26040C.
Thx for proposing sure something to watch

**Comment by Fearuncle** [comment]
Now 18

**Comment by FedeGueli** [comment]
it seems already slowing down as a cluster.
